<launch>

	<node pkg="rosbag" type="play" name="player" args="-r 0.01 -s 3 /home/viki/catkin_WS/src/ransac_project/bags/laser_scanner4.bag"/>

	<node pkg = "ransac_project" type = "ransac" name = "ransac" >
		<param name = "threshold" value = "0.7" />
		<param name = "p_inliers" value = "0.35" />
		<param name = "dataWidth" value = "3.0" />
	</node>
	
	<node pkg = "ransac_project" type = "ransac_control" name = "ransac_control" >
		<param name = "which_car" value = 'pioneer' />
		<param name = "v_linear" value = "0.3" />
		<param name = "lenght" value = "1.5" />
		<param name = "KPT" value = "20" />
		<param name = "KIT" value = "0.1" />
		<param name = "KRT" value = "1" />
		<param name = "KVT" value = "0.1" />
	</node>

</launch>
